packages/web3-shared/evm/assets/chains.json

```diff
@@ -335,6 +335,56 @@
         "chainId": 10,
         "networkId": 10
     },
+    {
+        "name": "Boba Network",
+        "chain": "BOBA",
```

Comment:
```suggestion
        "chain": "Boba",
```


Please make sure this button will not show "Wrong network"
![image](https://user-images.githubusercontent.com/1141198/151323099-5a747176-7252-494a-ba42-cf6fa436d174.png)